src/system_info/system_info_cellular_network_ivi.cc

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "system_info/system_info_cellular_network.h"
+
+SysInfoCellularNetwork::SysInfoCellularNetwork()
+    : status_(""),
+      apn_(""),
+      ipAddress_(""),
+      ipv6Address_(""),
+      mcc_(0),
+      mnc_(0),
+      cellId_(0),
+      lac_(0),
+      isRoaming_(false),
+      // FIXME(liying): NOT SUPPORTED
+      isFlightMode_(false),
```

Comment:
isFlightMode atrribute can not be got from ofono, and acording to [Tizen api spec] (https://developer.tizen.org/dev-guide/2.3.0/org.tizen.mobile.native.apireference/group__CAPI__SYSTEM__SYSTEM__SETTINGS__MODULE.html#ga56c8fa435516884c5648efecdd871eaa), there will be SYSTEM_SETTINGS_KEY_NETWORK_FLIGHT_MODE in ````#include <system_settings.h>````. So will support isFlightMode in the future as system_settings package does not have the feature now.